### PR TITLE
CVE-2011-3882 and CVE-2011-3956

### PR DESCRIPTION
**Pull request by ijlondon**

### CODE REVIEW

File: cves/CVE-2011-3882.yml [hunk 1]
```diff
@@ -93,10 +99,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit:
```
Comment (samshiffman321) [inline cves/CVE-2011-3882.yml]: no interesting commits?

File: cves/CVE-2011-3882.yml [hunk 2]
```diff
@@ -127,38 +133,38 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies: 
-    note: 
+    applies:
```
Comment (samshiffman321) [inline cves/CVE-2011-3882.yml]: this vulnerability should apply to at least one of the lessons we learned about in class

File: cves/CVE-2011-3956.yml [hunk 3]
```diff
@@ -15,31 +15,37 @@ description_instructions: |
...
+repo:
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: 69729cacd393118fa8e80fe30b8959df986435ab
   :note: ''
-vccs: []
+vccs:
+- :commit: d2f05d07a90cf5c5bad814b603175acaefa254b5]
```
Comment (samshiffman321) [inline cves/CVE-2011-3956.yml]: there should not be a ']' at the end of that hash

File: cves/CVE-2011-3956.yml [hunk 4]
```diff
@@ -92,10 +98,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit:
```
Comment (samshiffman321) [inline cves/CVE-2011-3956.yml]: no interesting commits?

File: cves/CVE-2011-3882.yml [hunk 5]
```diff
@@ -15,32 +15,37 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description:
+  Use-after-free vulnerability allows remote attackers to cause a denial of
+  service or possibly have unspecified other impact via vectors related to media
+  buffers.
```
Comment (ijlondon) [inline cves/CVE-2011-3882.yml]: I'll look into revising this, thanks

File: cves/CVE-2011-3882.yml (hunk 1 above)
Comment (ijlondon) [inline cves/CVE-2011-3882.yml]: I'll add some in

File: cves/CVE-2011-3882.yml (hunk 2 above)
Comment (ijlondon) [inline cves/CVE-2011-3882.yml]: I didn't see one that applied but I'll look into the the terms we never went over to see if it fits one of those

File: cves/CVE-2011-3956.yml (hunk 3 above)
Comment (ijlondon) [inline cves/CVE-2011-3956.yml]: good catch

**Comment by ijlondon** [comment]
Eric,
Thanks for the grammar and spelling catches, I'll make those changes. 
I'll also look into adding that note as well.

**Comment by noesterle** [comment]
CVE-2011-3882
    Spelling, Grammar, Style:
         Line 18 - include the possibility of running malicious code.
         Line 18 - cut down on the jargon in the description.
         Line 80, 92 - Fragment, consider making this a full sentence.
    Methodology or Investigation comments
         Line 80 - explain clusterfuzz. Is this a google tool? Is this freely available? How does this work? Don't forget, this is supposed to be aimed at people with minimal knowledge of coding practices.
         Line 181:  Could the fact that this wasn't found also be a testing issue?

CVE-2011-3956
    Spelling, Grammar, Style:
         Line 39 - remove "]" at the end of the VCC.
         Line 61, 79, 91, 139, 148, - Fragment, consider making this a full sentence.
    Methodology or Investigation comments
         Line 25 - Was there a bounty for this vulnerability? If not, I'd suggest filling in "0".

**Pull request merged**